fitbenchmarking/fitting/scipy/func_def.py

```diff
@@ -35,20 +34,8 @@ def function_definitions(problem):
     """
     problem_type = extract_problem_type(problem)
```

Comment:
Can you add a test for this new function?

Reply:
`scipy/func_def.py` currently does not have its test file. It will try to add that accordingly.